libs/datasets/dataset_utils.py

```diff
@@ -9,12 +9,17 @@
 
 from libs.us_state_abbrev import US_STATE_ABBREV
 
-if os.getenv("COVID_DATA_PUBLIC"):
-    LOCAL_PUBLIC_DATA_PATH = pathlib.Path(os.getenv("COVID_DATA_PUBLIC"))
-else:
-    LOCAL_PUBLIC_DATA_PATH = (
-        pathlib.Path(__file__).parent.parent / ".." / ".." / "covid-data-public"
-    )
+
+def _get_public_data_path():
+    """Sets global path to covid-data-public directory."""
+    if os.getenv("COVID_DATA_PUBLIC"):
```

Comment:
Is there a list of environmental variables that we attempt to access? I didn't realize that we had this as an option and have been doing some hacky jupyter lab %cd stuff to make it explicit in the notebook what working directory we need.

Reply:
We don't have a comprehensive place yet.  I would really like to use something like [pydantic settings](https://pydantic-docs.helpmanual.io/usage/settings/) eventually to keep all settings easily in one spot

Reply:
Yeah that's definitely an option.  

I have found that those end up getting out of date (definitely preventable) and having a single source of truth for constants is helpful (even if you're not using environment variables)